testdata/go/v3/memcached-operator/monitoring/metrics.go

```diff
@@ -0,0 +1,21 @@
+package monitoring
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	// MemcachedNumberOfReplicasReconcileCountTotal counts Memcached replicas number reconciliations
+	MemcachedNumberOfReplicasReconcileCountTotal = prometheus.NewCounter(
```

Comment:
```suggestion
	memcachedNumberOfReplicasReconcileCountTotal = prometheus.NewCounter(
```

Why do we need to export this one?
Where it should be used outside of here?

Reply:
We increment this metric in the controller. Please see https://github.com/operator-framework/operator-sdk/blob/b59d23e8b7e1e7dfa460e18b096856c1240372e7/testdata/go/v3/memcached-operator/controllers/memcached_controller.go#L108